e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -9,6 +9,7 @@ Commands:
   code-pushup autorun       Shortcut for running collect followed by upload
   code-pushup collect       Run Plugins and collect results
   code-pushup upload        Upload report results to the portal
+  code-pushup compare       Compare 2 report files and create a diff file
```

Comment:
While I see the `compare` command in `help`, I cannot see the required arguments here. I only see it when I run `code-pushup compare --help`. This does not seem intuitive to me. 
If not add them directly, I would create a new group for these commands to display them explicitly for the `compare` command.

Reply:
Why not, but I would optimize the option groups in #553 or once it's merged, so that it's done as a whole.